templates/config/openkm.properties

```diff
@@ -1,6 +1,6 @@
 # OpenKM Hibernate configuration values
 spring.datasource.driver-class-name=org.postgresql.Driver
-spring.datasource.url=jdbc:postgresql://localhost:{{ openkm_db_cont_port | mandatory }}/{{ openkm_db_name | mandatory }}
+spring.datasource.url=jdbc:postgresql://localhost:{{ postgres_ha_cont_port | mandatory }}/{{ openkm_db_name | mandatory }}
```

Comment:
Please don't do this. Roles should not mix variables between one another.

In order to pass this from PostgresHA role you need to provide it in the `group_vars` in the fleet repo.